news/2025/2025-02-27-introducing-teams.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: Introducing Teams
+date: 2025-02-27 12:00:00 +0000
+---
+
+These are not "clans", they're not "subdivisions", nor are they "guilds" — they are **teams**! And they're here to stay.
+
+![](/wiki/shared/news/2025-02-27-introducing-teams/banner.jpg)
+
+It's time for a [15-year-old feature request](https://osu.ppy.sh/community/forums/topics/29780) to see reality. This feature which was once requested by [[Sean]](https://osu.ppy.sh/users/202721) exactly 14 years, 9 months and 10 days ago, ended up amassing a total priority of **+11.744** over the years. An idea which was widely supported by the community up to this day has finally come into fruition!
+
+Despite how many years have passed since the inception of the idea, the core goals of the feature remain the same: to grow and nurture osu!'s many, scattered sub-communities. Users can find new ways to engage with other like-minded members with many of the new features to come. We always knew this was a community game, but this feature will make this aspect ever so slightly more prevalent!
+
+## What's new
+
+The [last osu!(lazer) update](https://osu.ppy.sh/home/news/2025-02-20-osulazer-updates-february-20-2025) introduced a new "Teams" feature. This is a new way for the community to express shared interests and proudly display groups they are part of.
+
+![](/wiki/shared/news/2025-02-27-introducing-teams/team-banner.jpg)
+
+Teams are shown besides usernames in places like user profiles, leaderboards, and forums. Each team gets a flag and name to display everywhere, a unique acronym to go along with it, a banner and description, and their own internal leaderboard.
+
+![](/wiki/shared/news/2025-02-27-introducing-teams/team-display.png)
+
+Users can interact with all of these features by going to the ["My Team"](https://osu.ppy.sh/teams) page at any time. Players not yet in a team will be given the option to create one. Alternatively, by going to the page of an existing team, users will be able to apply to any team they see fit!
+
+### Features
+
+While the feature is still in its infancy, there are plenty of things users can already do with teams:
+
+- Create and join other users' teams.
+- Manage incoming applications.
+- Compete with your peers in the team's leaderboard.
+- Proudly display your allegiance with a team flag and unique acronym!
+
+You can expect to see more features in the future as the system is refined, such as displaying team leaderboards in osu!(lazer) or team versus team matches.
+
+### Limitations
+
+Despite the many features already introduced from the get-go, there are still some limitations to keep in mind:
+
+- Team acronyms are unique.
+- Both team acronyms and names are permanent.
+- Team leaders cannot be swapped.
+- The maximum team size is dependent on the amount of [osu!supporters](/wiki/osu!supporter) on the team.
+  - Each supporter will add 4 slots to the base team size of 8, up to a cap of 256.
```

Comment:
I'm not sure this is a "limitation" as such. I guess it's fine to list here but it's not goign to change.

Reply:
Even if it's something that isn't going to change, that is indeed a limitation worth noting.

Reply:
Then the sentence above needs to change in line with this. It reads wrong as it stands.

Reply:
Hmm, I guess you are afraid people may take the comment below as to recommend switching the teams' max size. Can make it very clear it is not going to change then.